docs/hub/_toctree.yml

```diff
@@ -46,6 +46,8 @@
       title: Model Card Guidebook
     - local: model-card-landscape-analysis
       title: Landscape Analysis
+    - local: model-cards-components
+      title: Card Components
```

Comment:
is there anything else needed to add a section to the ToC? It doesn't show on https://moon-ci-docs.huggingface.co/docs/hub/pr_1003/en/model-cards (cc @mishig25 )

Reply:
Hey @krampstudio ,

right link would be: https://moon-ci-docs.huggingface.co/docs/hub/pr_1093/en/model-cards

and there the Card Components section is shown 🎉

![image](https://github.com/huggingface/hub-docs/assets/20651387/8d135aed-bd05-4908-8487-ae0f1f9f7b6f)